appinventor/components/src/com/google/appinventor/components/annotations/UsesAssets.java

```diff
@@ -22,5 +22,5 @@
   /**
    * The filenames of the required assets separated by commas.
    */
-  String fileNames() default "";
```

Comment:
We need to define a default value, in case a person has added an annotation without actually passing a String array. Simply just set default value to an empty array.

Reply:
...also make sure to apply default value for all your modified annotations.